### PR TITLE
Fixes some spelling mistakes

### PR DESCRIPTION
**Pull request by Aakash-kaushik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,11 +115,11 @@ Currently supported datasets are mentioned below :
 | --- | --- | --- |
 |  MNIST | DataLoader<>&nbsp;("mnist"); | MNIST dataset is the de facto “hello world” dataset of computer vision.<br/> Each image is 28 pixels in height and 28 pixels in width, for a total of 784 pixels in total. The first column, called "label", is the digit that was drawn by the user. The rest of the columns contain the pixel-values of the associated image.|
 |  Pascal VOC Detection | DataLoader<mat, field<vec>>&nbsp;("voc-detection") | The Pascal VOC challenge is a very popular dataset for building and evaluating algorithms for image classification, object detection and segmentation.<br/> VOC detection dataset provides support for loading object detection dataset in PASCAL VOC. Note : By default we refer to VOC - 2012 dataset as VOC dataset.|
-| CIFAR 10 | DataLoader<>&nbsp;("cifar10"); | The CIFAR-10 dataset consists of 60000 32x32 colour images in 10 classes, with 6000 images per class. There are 50000 training images and 10000 test images.|
+| CIFAR 10 | DataLoader<>&nbsp;("cifar10"); | The CIFAR-10 dataset consists of 60000 32x32 color images in 10 classes, with 6000 images per class. There are 50000 training images and 10000 test images.|
```
Comment (birm) [inline README.md:118]: I think this is correct either way. Do we use one more spelling more commonly than the other?

File: README.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ We provide support to load annotations represented in XML files and their corres
 #### 3. Preprocessing.
 
 For all datasets that we support we provide, We preprocess them internally. We also
-provide access to preprocessor functions for standard datasets incase one needs to
+provide access to preprocessor functions for standard datasets in-case one needs to
```
Comment (birm) [inline README.md:173]:
```suggestion
provide access to preprocessor functions for standard datasets in case one needs to
```

**Review by birm — COMMENTED** [review]
Thanks for looking over this repo's comments. I just have two observations

File: README.md (hunk 1 above)
Comment (Aakash-kaushik) [reply README.md:118]: Back to colour then. :D

**Review by kartikdutt18 — APPROVED** [review]

**Review by birm — APPROVED** [review]

**Pull request merged**